log/src/main/java/org/apache/felix/log/FormatterLoggerImpl.java

```diff
@@ -26,12 +26,13 @@
 
 public class FormatterLoggerImpl extends LoggerImpl implements FormatterLogger {
 
-    public FormatterLoggerImpl(
```

Comment:
This forces an version change. Why should this not be public?

Reply:
Is there an expectation that this would be usable outside the JAR itself? The `org.apache.felix.log` package is not exported at all.

Reply:
I am sorry, I am getting mixed signals here. Do you want me to put the `public` back?

Reply:
i would not change it. 
@laeubi  opinion?

Reply:
If it is not exported then it is effective private in OSGi, in any case I would add such change in a separate PR if desired and do not mix it with the usage of annotations.

Beside that it will only be a micro change.

Reply:
So please let it Public fot this PR

Reply:
Reverted the public removal.